Translations/InsaneShops_Russian.lang

```diff
@@ -1,91 +1,91 @@
-#GUI_SHOPCREATION_INCREASE_ACTION: Click to increase amount
-#GUI_SHOPCREATION_REDUCE_ACTION: Click to reduce amount
-#GUI_SHOPCREATION_UNAVAILABLE_EXPLANATION: This option is not available
-#GUI_SHOPCREATION_CONTINUE_EXPLANATION: Click to continue
-#GUI_SHOPSETTINGS_CONVERTADMINSHOP_TITLE: Convert to Admin Shop
-#GUI_SHOPSETTINGS_CONVERTADMINSHOP_ACTION: **Click** to convert
-#GUI_SHOPSETTINGS_STOCK_ACTION: Click to **manage** items
-#GUI_SHOPSETTINGS_STOCK_REVENUE_IF_SOLD: Total revenue if sold
-#GUI_SHOPSETTINGS_TRANSACTIONS_ACTION: Click to **view** transactions
-#GUI_SHOPSETTINGS_DELETE_TITLE: Close Shop
-#GUI_SHOPSETTINGS_DELETE_ACTION: Click to close shop
-#GUI_SHOPSETTINGS_DELETE_NEEDWITHDRAWING: Please withdraw all stocked items first
-#GUI_STOCKEDVIEW_TITLE: Items in Stock
-#GUI_STOCKEDVIEW_ACTION: Click to **move** into your Inventory
-#GUI_STOCKEDVIEW_INSTRUCTION: Add items by clicking on them in your Inventory
-#GUI_TRANSACTIONS_PURCHASETIME: Purchased **%1** ago
-#SHOP_CREATION_NOPERMISSION: You are not allowed to create a Shop
-#SHOP_CREATION_INVALID_ITEM: You cant sell this item
-#SHOP_CREATION_REACHED_LIMIT: You are not allowed to create more than **%1** Shops
-#ACTIONBAR_NOTICES_OPENSHOP: **Right Click** to open this **Shop**
-#ACTIONBAR_NOTICES_CREATESHOP: **Sneak + Left Click** to create a new Shop
-#MESSAGE_BUY_INVFULL: Could not buy **%1**. Your inventory is full.
-#MESSAGE_BUY_COMPLETED: Successfully bought **%1x %2 **for **%3
+SHOP: Магазин
+SHOPS: Магазины
+BUY: Покупка
+TYPEIN: Ввести
+ITEMS: товара(ов)
+SELL: Продать
+REVENUE: Прибыль
+TRANSACTIONS: Транзакции
+PURCHASING: Покупка
+BUYING: Покупка
+SELLING: Продажа
+PRICE: Цена
+UNITS: Единицы
+AMOUNT: Количество
+BUY_FOR: Покупка:
+SELL_FOR: Продажа:
+AVAILABLE: В наличии
+PRICE_PER_ITEM: Цена за штуку
+STOCKED_ITEMS: Товары
+INCREASE: Увеличить
+DECREASE: Уменьшить
+QUANTITY: Количество
+CONFIRM: Подтвердить
+FOR: за
+YOU_WILL_RECEIVE: Вы получите
+ENCHANTED: Enchanted
+ITEMS_AVAILABLE: В наличии
+CONTINUE: Продолжить
+MONEY_PAID: Выплачено
+GUI_OVERVIEW_OPENSHOPS_ACTION: Нажмите для открытия магазинов
+GUI_OVERVIEW_OPENSETTINGS_ACTION: Нажмите для открытия настроек
+GUI_SHOPSETTINGS_BUY_LEFTACTION: **ЛКМ** для **изменения** цены покупки
+GUI_SHOPSETTINGS_BUY_QACTION: **Q** для выключения покупки
+GUI_SHOPSETTINGS_BUY_OTHERACTION: **Нажмите** для включения **покупки**
+GUI_SHOPSETTINGS_BUY_EXPLANATION: Покупка **%1**/штуку
+GUI_SHOPSETTINGS_SELL_LEFTACTION: **ЛКМ** для **изменения** цены продажи
+GUI_SHOPSETTINGS_SELL_QACTION: **Q** для выключения продажи
+GUI_SHOPSETTINGS_SELL_OTHERACTION: **Нажмите** для включения **продажи**
+GUI_SHOPSETTINGS_SELL_EXPLANATION: Продажа **%1**/штуку
+GUI_MONEYSELECTOR_TYPEIN_ACTION: **Нажмите** для ввода цены
+GUI_MONEYSELECTOR_INCREASE_ACTION: **ЛКМ** для добавления **%1**
+GUI_MONEYSELECTOR_DECREASE_ACTION: **ЛКМ** для удаления **%1**
+GUI_MONEYSELECTOR_SWITCHUNIT: **Q******** для переключения единиц
```

Comment:
That looks like too many symbols to me

Reply:
Its okay, because some world in translation is unused, and I just delete this world to save color, because in this string if not add a lot of on the end "Q" will not be colored